atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -305,6 +304,10 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
                 managementSupplier,
                 atlasFactory.getTimestampStoreInvalidator(),
                 userAgent());
+        TimestampCorroboratingTimelockService corroboratingTimelockService = TimestampCorroboratingTimelockService
+                .create(rawLockAndTimestampServices.timelock());
+        LockAndTimestampServices lockAndTimestampServices =
+                withCorroboratingTimestampService(rawLockAndTimestampServices, corroboratingTimelockService);
```

Comment:
Non-actionable: Technically there is a change in behaviour here, because we've changed the wrapping order - in particular, before this change the check is instrumented (so will show up as part of the time taken for a getFreshTimestamps) while now only the inner call is instrumented. I think this is fine.